tests/test_connection.py

```diff
@@ -90,6 +90,34 @@ async def test_basic_authentication() -> None:
         await couchdb.check_credentials()
 
 
+async def test_utf8_basic_authentication(couchdb: CouchDB) -> None:
+    import os
+
+    users = await couchdb.create("_users", exists_ok=True)
+    doc = await users.create("org.couchdb.user:aiocouch_test_user_utf8")
+    doc["name"] = "aiocouch_test_user_utf8"
+    doc["password"] = "passwor§"
+    doc["roles"] = ["aiocouch_test_role"]
+    doc["type"] = "user"
+    await doc.save()
+    assert doc["_id"] == "org.couchdb.user:aiocouch_test_user_utf8"
+    assert doc.id == "org.couchdb.user:aiocouch_test_user_utf8"
```

Comment:
I'd rather see this as another fixture akin to `couchdb_user_account` in `conftest.py`